packages/apps/tools/src/pages/faucet/new/index.tsx

```diff
@@ -325,6 +325,7 @@ const NewAccountFaucetPage: FC = () => {
                     });
                   }
                 }}
+                aria-label="Add Public Key"
```

Comment:
@sanderlooijenga @KristinaSpasevska I added a label here so I can identify the button.